src/Commands/StartFrankenPhpCommand.php

```diff
@@ -103,7 +103,9 @@ public function handle(ServerProcessInspector $inspector, ServerStateFile $serve
             'CADDY_SERVER_LOGGER' => 'json',
             'CADDY_SERVER_SERVER_NAME' => $serverName,
             'CADDY_SERVER_WORKER_COUNT' => $this->workerCount() ?: '',
+            'CADDY_SERVER_WORKER_DIRECTIVE' => $this->workerCount() ? "num {$this->workerCount()}" : '',
```

Comment:
Having `CADDY_SERVER_WORKER_DIRECTIVE` in addition to the previous `CADDY_SERVER_WORKER_COUNT` allows for backwards compatibility if users have their own previous version of the Caddyfile.

The new variable is needed because the `num` directive must have a value provided.